mergify_engine/actions/update.py

```diff
@@ -46,12 +77,9 @@ async def run(ctxt: context.Context, rule: rules.EvaluatedRule) -> check_api.Res
             )
         else:
             await signals.send(
-                ctxt.repository,
-                ctxt.pull["number"],
-                "action.update",
-                signals.EventNoMetadata(),
-                rule.get_signal_trigger(),
+                ctxt, "action.rebase", {"bot_account": bool(self.config["bot_account"])}
```

Comment:
Changing from `update` to `rebase` ? Aren't the `update` and `rebase` 2 different behaviors ?